H3 does not build with PG12

```
make[3]: Leaving directory '/tmp/tmptWgIDZ/h3-3.4.1/libh3-v3.4.4'
make[3]: Entering directory '/tmp/tmptWgIDZ/h3-3.4.1/libh3-v3.4.4'
[ 95%] Building C object CMakeFiles/h3ToHier.dir/src/apps/miscapps/h3ToHier.c.o
[ 96%] Building C object CMakeFiles/h3ToHier.dir/src/apps/applib/lib/kml.c.o
[ 97%] Building C object CMakeFiles/h3ToHier.dir/src/apps/applib/lib/utility.c.o
[ 98%] Building C object CMakeFiles/h3ToHier.dir/src/apps/applib/lib/test.c.o
[100%] Linking C executable bin/h3ToHier
make[3]: Leaving directory '/tmp/tmptWgIDZ/h3-3.4.1/libh3-v3.4.4'
[100%] Built target h3ToHier
make[2]: Leaving directory '/tmp/tmptWgIDZ/h3-3.4.1/libh3-v3.4.4'
Install the project...
-- Install configuration: ""
-- Installing: /tmp/tmptWgIDZ/h3-3.4.1/libh3-v3.4.4/install/lib/libh3.a
-- Installing: /tmp/tmptWgIDZ/h3-3.4.1/libh3-v3.4.4/install/bin/geoToH3
-- Installing: /tmp/tmptWgIDZ/h3-3.4.1/libh3-v3.4.4/install/bin/h3ToComponents
-- Installing: /tmp/tmptWgIDZ/h3-3.4.1/libh3-v3.4.4/install/bin/h3ToGeo
-- Installing: /tmp/tmptWgIDZ/h3-3.4.1/libh3-v3.4.4/install/bin/h3ToGeoBoundary
-- Installing: /tmp/tmptWgIDZ/h3-3.4.1/libh3-v3.4.4/install/bin/hexRange
-- Installing: /tmp/tmptWgIDZ/h3-3.4.1/libh3-v3.4.4/install/bin/kRing
-- Installing: /tmp/tmptWgIDZ/h3-3.4.1/libh3-v3.4.4/install/bin/h3ToGeoBoundaryHier
-- Installing: /tmp/tmptWgIDZ/h3-3.4.1/libh3-v3.4.4/install/bin/h3ToGeoHier
-- Installing: /tmp/tmptWgIDZ/h3-3.4.1/libh3-v3.4.4/install/bin/h3ToHier
-- Installing: /tmp/tmptWgIDZ/h3-3.4.1/libh3-v3.4.4/install/include/h3/h3api.h
-- Installing: /tmp/tmptWgIDZ/h3-3.4.1/libh3-v3.4.4/install/lib/cmake/h3/h3Config.cmake
-- Installing: /tmp/tmptWgIDZ/h3-3.4.1/libh3-v3.4.4/install/lib/cmake/h3/h3ConfigVersion.cmake
-- Installing: /tmp/tmptWgIDZ/h3-3.4.1/libh3-v3.4.4/install/lib/cmake/h3/h3Targets.cmake
-- Installing: /tmp/tmptWgIDZ/h3-3.4.1/libh3-v3.4.4/install/lib/cmake/h3/h3Targets-noconfig.cmake
make[1]: Leaving directory '/tmp/tmptWgIDZ/h3-3.4.1/libh3-v3.4.4'
sed -e 's/@EXTVERSION@/3.4.1/g' \
        -e 's/@LIBH3_REQUIRED_MAJOR@/3/g' \
        -e 's/@LIBH3_REQUIRED_MINOR@/4/g' \
        -e 's/@LIBH3_REQUIRED_PATCH@/0/g' \
        src/extension.in.h > src/extension.h
gcc -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Werror=vla -Wendif-labels -Wmissing-format-attribute -Wformat-security -fno-strict-aliasing -fwrapv -fexcess-precision=standard -Wno-format-truncation -Wno-stringop-truncation -g -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -fno-omit-frame-pointer -fPIC -Ilibh3-v3.4.4/install/include -I. -I./ -I/usr/include/postgresql/12/server -I/usr/include/postgresql/internal  -Wdate-time -D_FORTIFY_SOURCE=2 -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include/mit-krb5  -c -o src/hierarchy.o src/hierarchy.c
gcc -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Werror=vla -Wendif-labels -Wmissing-format-attribute -Wformat-security -fno-strict-aliasing -fwrapv -fexcess-precision=standard -Wno-format-truncation -Wno-stringop-truncation -g -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -fno-omit-frame-pointer -fPIC -Ilibh3-v3.4.4/install/include -I. -I./ -I/usr/include/postgresql/12/server -I/usr/include/postgresql/internal  -Wdate-time -D_FORTIFY_SOURCE=2 -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include/mit-krb5  -c -o src/miscellaneous.o src/miscellaneous.c
gcc -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Werror=vla -Wendif-labels -Wmissing-format-attribute -Wformat-security -fno-strict-aliasing -fwrapv -fexcess-precision=standard -Wno-format-truncation -Wno-stringop-truncation -g -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -fno-omit-frame-pointer -fPIC -Ilibh3-v3.4.4/install/include -I. -I./ -I/usr/include/postgresql/12/server -I/usr/include/postgresql/internal  -Wdate-time -D_FORTIFY_SOURCE=2 -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include/mit-krb5  -c -o src/type.o src/type.c
gcc -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Werror=vla -Wendif-labels -Wmissing-format-attribute -Wformat-security -fno-strict-aliasing -fwrapv -fexcess-precision=standard -Wno-format-truncation -Wno-stringop-truncation -g -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -fno-omit-frame-pointer -fPIC -Ilibh3-v3.4.4/install/include -I. -I./ -I/usr/include/postgresql/12/server -I/usr/include/postgresql/internal  -Wdate-time -D_FORTIFY_SOURCE=2 -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include/mit-krb5  -c -o src/uniedges.o src/uniedges.c
gcc -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Werror=vla -Wendif-labels -Wmissing-format-attribute -Wformat-security -fno-strict-aliasing -fwrapv -fexcess-precision=standard -Wno-format-truncation -Wno-stringop-truncation -g -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -fno-omit-frame-pointer -fPIC -Ilibh3-v3.4.4/install/include -I. -I./ -I/usr/include/postgresql/12/server -I/usr/include/postgresql/internal  -Wdate-time -D_FORTIFY_SOURCE=2 -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include/mit-krb5  -c -o src/extension.o src/extension.c
gcc -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Werror=vla -Wendif-labels -Wmissing-format-attribute -Wformat-security -fno-strict-aliasing -fwrapv -fexcess-precision=standard -Wno-format-truncation -Wno-stringop-truncation -g -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -fno-omit-frame-pointer -fPIC -Ilibh3-v3.4.4/install/include -I. -I./ -I/usr/include/postgresql/12/server -I/usr/include/postgresql/internal  -Wdate-time -D_FORTIFY_SOURCE=2 -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include/mit-krb5  -c -o src/indexing.o src/indexing.c
src/indexing.c: In function ‘h3_to_geo_boundary’:
src/indexing.c:79:21: error: ‘M_PI’ undeclared (first use in this function); did you mean ‘P_PID’?
       delta = - 2 * M_PI;
                     ^~~~
                     P_PID
src/indexing.c:79:21: note: each undeclared identifier is reported only once for each function it appears in
src/indexing.c:89:23: warning: implicit declaration of function ‘fabs’ [-Wimplicit-function-declaration]
         if (extend && fabs(lon - firstLon) > M_PI) // check if different sign
                       ^~~~
src/indexing.c:89:23: warning: incompatible implicit declaration of built-in function ‘fabs’
src/indexing.c:89:23: note: include ‘<math.h>’ or provide a declaration of ‘fabs’
src/indexing.c:23:1:
+#include <math.h>

src/indexing.c:89:23:
         if (extend && fabs(lon - firstLon) > M_PI) // check if different sign
                       ^~~~
make: *** [<builtin>: src/indexing.o] Error 1
ERROR: command returned 2: make PG_CONFIG=/usr/bin/pg_config all
```

--- a/src/indexing.c
+++ b/src/indexing.c
@@ -20,6 +20,7 @@
 
 #include <h3/h3api.h> // Main H3 include
 #include "extension.h"
+#include <math.h>
 
 // Indexes the location at the specified resolution
 PG_FUNCTION_INFO_V1(h3_geo_to_h3);

